app/storybook/.storybook/preview.tsx

```diff
@@ -11,18 +11,25 @@ import './global.css'
 const GlobalStyles = () => {
   const theme = useTheme()
   if (document && theme) {
-    document.body.style.setProperty('--bg-color', theme.bgSecondary as string)
+    document.body.style.setProperty('--bg-color', theme.backgroundSecondary as string)
   }
   return null
 }
 
+// const MyCustomButton = ({ children }: React.ComponentPropsWithoutRef<'button'>) => <button>{children}</button>
```

Comment:
Well, this is a spoiler of the upcoming feature where customers can replace primitive components with their own. The mechanism is there but I decided to postpone the official introduction of this feature due to the large size of the current PR. I left this comment as a reminder. I think it's okay, since it won't affect our customers anyway.